src/libs/ReportUtils.ts

```diff
@@ -4084,6 +4086,25 @@ function getParticipantsIDs(report: OnyxEntry<Report>): number[] {
     return participants;
 }
 
+/**
+ * Returns an array of the visible member accountIDs for a report*
+ */
+function getVisibleMemberIDs(report: OnyxEntry<Report>): number[] {
+    if (!report) {
+        return [];
+    }
+
+    const visibleChatMemberAccountIDs = report.visibleChatMemberAccountIDs ?? [];
+
+    // Build participants list for IOU/expense reports
+    if (isMoneyRequestReport(report)) {
+        const onlyTruthyValues = [report.managerID, report.ownerAccountID, ...visibleChatMemberAccountIDs].filter(Boolean) as number[];
+        const onlyUnique = [...new Set([...onlyTruthyValues])];
+        return onlyUnique;
```

Comment:
NAB, I don't _think_ these checks are necessary. If the `visibleChatMemberAccountIDs` is wrong then we have a problem somewhere and shouldn't cover it up.

Reply:
I think it might be possible to eventually have duplicates, but the truth check I agree we should remove.

I think we can remove the Set once we make the backend guarantee uniqueness.

Reply:
Personally, I would just trust that it is correct. If the manager or owner are not in the `visibleChatMemberAccountIDs` then it would suggest to me some problem in some other layer.